src/main/java/uk/nhs/digital/nhsconnect/nhais/inbound/RegistrationConsumerService.java

```diff
@@ -56,8 +57,10 @@ public void handleRegistration(InboundMeshMessage meshMessage) {
 
         Streams.zip(inboundStateRecords.stream(), supplierQueueDataToSend.stream(), Pair::of)
             .forEach(pair -> {
-                inboundGpSystemService.publishToSupplierQueue(pair.getRight());
-                inboundStateRepository.save(pair.getLeft());
+                if (!pair.getRight().getTransactionType().equals(ReferenceTransactionType.Inbound.CLOSE_QUARTER_NOTIFICATION)) {
+                    inboundGpSystemService.publishToSupplierQueue(pair.getRight());
+                    inboundStateRepository.save(pair.getLeft());
```

Comment:
I think that inbound state repository should be updated

Reply:
done